packages/core/src/types.ts

```diff
@@ -26,6 +26,7 @@ export interface WalletConfig {
     type: string
     [key: string]: unknown
   }
+  masterSecretId?: 'string'
```

Comment:
Maybe I'm missing something but I think it should be string without quotes
```suggestion
  masterSecretId?: string
```

Reply:
Yes, this only allows the string literal `string`. Any other string is not allowed.